flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/table/CalcTest.scala

```diff
@@ -138,6 +138,57 @@ class CalcTest extends TableTestBase {
 
     util.verifyTable(resultTable, expected)
   }
+
+  @Test
+  def testAddColumns(): Unit = {
```

Comment:
What is the difference between all those cases? Do we parametrize them with `(isBatch, String/Expression)`. We should think of unifying it. It does not make sense to copy it to all places.

Reply:
You are right, the only difference is batch or stream. I think this is a common issue, not only the column test case. can we unify this in the other PR?  if you want we can add all test in the stream, and remove all of batch test case. what do you think?